src/SourceBuild/content/build.proj

```diff
@@ -26,7 +26,12 @@
     <ProjectReference Include="$(RepositoryEngineeringDir)publish.proj" />
   </ItemGroup>
 
-  <Target Name="PrintInfo">
+  <ItemGroup Condition="'$(DotNetBuildPass)' == 'final'">
+    <ProjectReference Include="$((RepositoryEngineeringDir))join-verticals.proj" />
```

Comment:
```suggestion
    <ProjectReference Include="$(RepositoryEngineeringDir)join-verticals.proj" />
```

Reply:
my bad, messed this up when I was copying